lms/djangoapps/grades/rest_api/v1/utils.py

```diff
@@ -40,6 +41,21 @@ def get_page_size(self, request):
 
         return self.page_size
 
+    def get_paginated_response(self, data, **kwargs):  # pylint: disable=arguments-differ
+        """
+        Return a response given serialized page data and kwargs. Each key-value pair is added to the response.
+        """
+        resp = OrderedDict([
+            ('next', self.get_next_link()),
+            ('previous', self.get_previous_link()),
+            ('results', data)
+        ])
```

Comment:
48-52 look like what we'd get if we called super's version of this method and pulled `data` out of the Response.

My understanding is it may make more sense to call super, mutate the `data` attached, and then return the same Response. Makes (slightly) fewer assumptions about how the method definition we're overriding might change in the future

Reply:
That makes sense to me. Thanks for catching that.